app/components/CommaSeparateItems/index.tsx

```diff
@@ -6,8 +6,8 @@ import { joinList } from '#utils/common';
 import styles from './styles.css';
 
 interface Item {
-    id: number;
-    name: string;
+    id: string;
+    displayName?: string | null | undefined;
```

Comment:
Instead of changing `name` to `displayName` field here, perform data transformation where CommaSeparateItems is used. However, if we want to change the field name, we need to ensure it is changed everywhere(where the component is used).